src/components/ImageAttribution.vue

```diff
@@ -2,6 +2,7 @@
   <section class="sidebar_section">
     <div>
       <div class="photo-attribution">
+        <h5>Image attribution</h5>
```

Comment:
You need to apply the `b-header` class here https://cc-vocabulary.netlify.com/?path=/docs/tokens-typography--figma#headings